### PR TITLE
Add eswiki badwords+informals to cawiki

### PR DESCRIPTION
**Pull request by kevinbazira**
Added Spanish badwords/informals to Catalan model and rebuilt Catalan model

### CODE REVIEW

**Comment by halfak** [comment]
```
Encountered 2 file(s) that should have been pointers, but weren't:
	models/cawiki.damaging.gradient_boosting.model
	models/cawiki.goodfaith.gradient_boosting.model
```

It looks like these files aren't in git-lfs.  This should have just happened automatically.  I'm looking into why it didn't.

File: requirements.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ mwapi >= 0.5.1, < 0.5.999
 mwtypes >= 0.3.0, < 0.3.999
 mwreverts >= 0.1.0, < 0.1.999
 mwxml >= 0.3.3
-revscoring >= 2.5.1, < 2.5.999
+revscoring >= 2.6.1 < 2.6.999
```
Comment (halfak) [inline requirements.txt]: Missing a comma here.

**Comment by halfak** [comment]
OK.  It looks like we get a mild effect from tracking spanish badwords/informals.  In general, it looks like the model does learn that the presence of spanish badwords/informals is bad.  We can test this using our cache injection.

Here, I've taken an edit that gets a borderline score (damaging proba: 0.286).  And I've re-scored the edit with 3 badwords and 3 informal words.  We can see that the damaging proba goes up to 0.349 and 0.333 respectively.  This suggest that the model has learned roughly what we expected it to. 

```
$ revscoring score models/cawiki.damaging.gradient_boosting.model --host=https://ca.wikipedia.org 22031565
22031565	{"prediction": false, "probability": {"false": 0.7144906187889479, "true": 0.2855093812110521}}
$ revscoring score models/cawiki.damaging.gradient_boosting.model --host=https://ca.wikipedia.org 22031565 --cache='{"datasource.spanish.badwords.revision.matches": ["mierda", "puta", "mierda"]}'
22031565	{"prediction": false, "probability": {"false": 0.6512222365571363, "true": 0.34877776344286376}}
$ revscoring score models/cawiki.damaging.gradient_boosting.model --host=https://ca.wikipedia.org 22031565 --cache='{"datasource.spanish.informals.revision.matches": ["copien", "esq", "asi"]}'
22031565	{"prediction": false, "probability": {"false": 0.6669917191659525, "true": 0.3330082808340475}}

**Comment by halfak** [comment]
I think we should take the parameters from the tuning reports and update the cawiki models.  It's a little different process for the editquality repo.  I'll update the PR.

**Comment by halfak** [comment]
I just re-built this branch with the models in LFS.  I think this is good to go.  @kevinbazira, could you take another look and merge if you think it all looks right?

**Comment by kevinbazira** [comment]
Looks like the CI build is still failing.

**Pull request merged**